src/dashboards/components/dashboard_index/DashboardsTableEmpty.tsx

```diff
@@ -32,18 +36,48 @@ const DashboardsTableEmpty: FC<ComponentProps> = ({
   }
 
   return (
-    <EmptyState size={ComponentSize.Large} testID="empty-dashboards-list">
-      <EmptyState.Text>
-        Looks like you don't have any <b>Dashboards</b>, why not create one?
-      </EmptyState.Text>
-      <AddResourceDropdown
-        onSelectNew={onCreateDashboard}
-        onSelectImport={summonImportOverlay}
-        onSelectTemplate={summonImportFromTemplateOverlay}
-        resourceName="Dashboard"
-        canImportFromTemplate={ALLOW_IMPORT_FROM_TEMPLATE}
-      />
-    </EmptyState>
+    <GoogleOptimizeExperiment
+      experimentID="9H85dx92QTylCNHIL9Y5Sg"
+      original={
+        <EmptyState size={ComponentSize.Large} testID="empty-dashboards-list">
+          <EmptyState.Text>
+            Looks like you don't have any <b>Dashboards</b>, why not create one?
+          </EmptyState.Text>
+          <AddResourceDropdown
+            onSelectNew={onCreateDashboard}
+            onSelectImport={summonImportOverlay}
+            onSelectTemplate={summonImportFromTemplateOverlay}
+            resourceName="Dashboard"
+            canImportFromTemplate={ALLOW_IMPORT_FROM_TEMPLATE}
+          />
+        </EmptyState>
+      }
+      variants={[
+        <GetResources
+          resources={[ResourceType.Buckets]}
+          key="9H85dx92QTylCNHIL9Y5Sg-1"
```

Comment:
What is this key for?

Reply:
Because it's an array of elements react complains if they don't have a key.

Reply:
oh but in that case, the key should be unique for each element, right?

Reply:
oh I see in this case there is only 1 element in the array. Can we maybe change the `key` value to be something else? It looks suspicious being that random string. Even just a `1` should do